pre-flight-checks.sh

```diff
@@ -8,8 +8,8 @@ ERRORS=()
 
 echo " *** Checking that site can be built."
 
-wok --serve >/dev/null 2>&1 &
-WOK_PID=$!
+python manage.py serve >/dev/null 2>&1 &
```

Comment:
Yes we could... it's not a priority for me right now though (although it'd be a good excuse to learn my way around the checks framework).